docs/source/releasenotes/add-ons/3.0.rst

```diff
@@ -34,7 +34,9 @@ Bug Fixes
 
 - [KC-6380] - CSV Parser Incorrectly Parsing Pipe-delimited file
 
-
+3.0.6
+^^^^^
+- [KC-5588] - Update S3 Source To Work Without Files In A Directory
```

Comment:
This isn't a bug fix it's an improvement/new feature

Reply:
Brenna, I thought we decided to just listing the stories in the addon release underneath the addon release number. What are you expecting to see here?

Reply:
I thought we did too but up above it says "Bug Fixes". I would remove the words "Bug fix" and move the Azure store down to the 3.0.6 list.

Reply:
moved the KC-5588 to the new feature under the 3.7 release notes - should it be removed from the addons page?

Reply:
No they should both be on the addons page as new features under 3.0.6.

Also please update the title to: Update S3 Source to no longer require folder and add region as a field.

The old title wasn't very accurate. I should have updated it before.